crates/ruff_server/README.md

```diff
@@ -1 +1,10 @@
-## The Ruff Language Server
+## `ruff server`
```

Comment:
nit: I have a slight preference for the former "Ruff Language Server" :)

Reply:
You know what? Same. Let's change it back 😄